tests/integration/cloud/providers/profitbricks.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+'''
+    :codeauthor: :email:`Ethan Devenport <ethand@stackpointcloud.com>`
+'''
+
+# Import Python Libs
+from __future__ import absolute_import
+import os
+import random
+import string
+
+# Import Salt Testing Libs
+from salttesting import skipIf
+from salttesting.helpers import ensure_in_syspath, expensiveTest
+
+ensure_in_syspath('../../../')
+
+# Import Salt Libs
+import integration
+from salt.config import cloud_providers_config
+from salt.ext.six.moves import range
+
+# Import Third-Party Libs
+try:
```

Comment:
This import block should be removed (since the profibricks driver doesn't use libcloud) and should be replaced with on that is looking for the profitbricks import defined in the driver:
```
+try:
 +    from profitbricks.client import ProfitBricksService
 +    from profitbricks.client import Server, NIC, Volume
 +    HAS_PROFITBRICKS = True
 +except ImportError:
 +    HAS_PROFITBRICKS = False
```